### DISCUSSION TITLE
CSS is not working on my index.html page. But other css files are working on non-main pages.

### DISCUSSION THREAD

**Discussion by AgnivC** [community/community#134969] [Programming Help]
### Select Topic Area

Question

### Body

Pretty much what's been said in the title. My Github repository is linked [here](https://github.com/AgnivC/paintings-using-css). I cannot figure out why "style.css" will not load for the main "index.html", and changes that I have tried on my repo don't seem to work either. Where am I going wrong here?

**Comment by davevad93** [comment]
Hi @AgnivC ,

I've checked the `index.html` and `style.css` files, your css file works because I've tested it with the browser dev tools by changing the body `background-color` property and the `h1` tag `font-size`, the problem is that you have many non-existent classes for example:

1. `.home-page-layout`
2. `.grid-container`
3. `.artist-details-container`
4. `.painting-links-container`

The four classes above are present in your `css` but don't actually exist in your `html` file. For instance, if you change the class `.painting-links-container` to `.painting-links` which is included in your `index.html` the result is the following:

![1](https://github.com/user-attachments/assets/0c251405-28ac-4c34-b7d4-b50b0b6545df)

**Comment by Ali-Iqbal-0** [comment]
hello sir you Have not added classes names with your divs thats why you getting error on it and you are using unused code in your css

**Comment by hojjatfazayeli** [comment]
Hi @AgnivC 
I checked your repository in github . your solution is add your css class on elements in html file. if you want see your style on html file you should doing this steps :
1- link your css file in head tag on html file.
2- use css class on html elements for see styles on html file.

Good Luck
